jenkins_jobs/trigger_weekly_host_os_build/script.sh

```diff
@@ -77,7 +77,7 @@ create_release_notes() {
     python host_os.py \
            --verbose \
            build-release-notes \
-               --packages-metadata-repo-url "$VERSIONS_REPOSITORY_URL" \
+               --packages-metadata-repo-url "$VERSIONS_PUSH_URL" \
```

Comment:
That's right, `update-versions` fetches from the upstream repo, but it creates a new commit and pushes it to `VERSIONS_PUSH_URL`. This commit is then used to create the pull request. It will be up-to-date because it was based on the upstream's master, not on the fork's master.